edx_proctoring/constants.py

```diff
@@ -46,7 +46,7 @@
 SOFTWARE_SECURE_CLIENT_TIMEOUT = (
     settings.PROCTORING_SETTINGS['SOFTWARE_SECURE_CLIENT_TIMEOUT'] if
     'SOFTWARE_SECURE_CLIENT_TIMEOUT' in settings.PROCTORING_SETTINGS
-    else getattr(settings, 'SOFTWARE_SECURE_CLIENT_TIMEOUT', 30)
+    else getattr(settings, 'SOFTWARE_SECURE_CLIENT_TIMEOUT', 60)
 )
 
 SOFTWARE_SECURE_SHUT_DOWN_GRACEPERIOD = (
```

Comment:
Can we add a comment to describe what this is?  Also, was it meant to be 2 heartbeats (e.g. 2x5s)?  Would it not be DEFAULT_CLIENT_POLLING_INTERVAL + REDUCED_CLIENT_POLLING_INTERVAL, or 30s + 15s = 45s?  Does it need a little extra wiggle room, like fixed 5-10s, or 3 heartbeats of DEFAULT_CLIENT_POLLING_INTERVAL + 2xREDUCED_CLIENT_POLLING_INTERVAL, or 30s + 2x15s = 60s.